client/src/main/scala/com/dwolla/circe/DurationAsSecondsCodec.scala

```diff
@@ -0,0 +1,10 @@
+package com.dwolla.circe
+
+import java.time.Duration
+
+import io.circe.{Decoder, Encoder}
+
+trait DurationAsSecondsCodec {
```

Comment:
I think it might be useful to expose objects that extend these traits so these implicits can be imported if that's more convenient than extending these traits. e.g.

```scala
object DurationAsSecondsCodec extends DurationAsSecondsCodec
```

so code that wants the imports can do either of these:

```scala
object UseCodec extends DurationAsSecondsCodec {
  implicitly[Codec[Duration]]
}

// or

import DurationAsSecondsCodec._
object UseCodec {
  implicitly[Codec[Duration]]
}
```

I _don't_ think we want to do this, though:

```scala
package object com.dwolla.circe extends DurationAsSecondsCodec with NullAsEmptyListCodec with …
```

These codecs are pretty specific, so I think we want them to be imported individually, and not as a consequence of doing something like this, for example:

```scala
import com.dwolla.circe._
```

Reply:
Good idea. Addressed.